.github/workflows/test-kitchen.yml

```diff
@@ -106,22 +106,12 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v3
     - name: Setup ruby
-      uses: actions/setup-ruby@v1.1.3
-    - name: Cache gems
-      uses: actions/cache@v3
+      uses: ruby/setup-ruby@v1
       with:
-        path: vendor/bundle
-        key: bundle-${{ runner.OS }}-${{ hashFiles('Gemfile.lock') }}
-        restore-keys: |
-          bundle-${{ runner.OS }}-
+        bundler-cache: true
     - name: Disable apparmor for mysqld
       run: |
         sudo apt-get install apparmor-utils
         sudo aa-disable /usr/sbin/mysqld
-    - name: Install gems
-      run: |
-        gem install bundler
-        bundle config set deployment true
```

Comment:
Do we need to preserve this? There are details for how to replace `bundle config` in the `setup-ruby` doco but it might not matter in this case as I think this might be to do with our caching?

Reply:
Look like this was a translation from using `--deployment` with bundler 1.x and that in turn was probably just copied from the example so this is probably fine to go as well.